kits/mou-drafter/apps/public/swiftlatex/index.html

```
@@ -0,0 +1,210 @@
+        }
+    </style>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.3/gh-fork-ribbon.min.css" />
+    <!-- Favicon
+    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+    <link rel="icon" type="image/png" href="favicon.png">
+</head>
+<body class="code-snippets-visible">
+<a class="github-fork-ribbon" href="https://github.com/SwiftLaTeX/SwiftLaTeX/" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
+<!-- Primary Page Layout
+–––––––––––––––––––––––––––––––––––––––––––––––––– -->
+<section class="container">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </section> ], start tag match failed [ <section class="container"> ] on line 46.

(tag-pair)